libraries/config/include/config/config.hpp

```diff
@@ -100,6 +100,8 @@ struct FullNodeConfig {
   std::vector<logger::Config> log_configs;
   bool is_light_node = false;       // Is light node
   uint64_t light_node_history = 0;  // Number of periods to keep in history for a light node
+  uint32_t dag_expiry_limit = kDagExpiryLevelLimit;
+  uint32_t max_levels_per_period = kMaxLevelsPerPeriod;
```

Comment:
They don't get values from config file. Why not using value kDagExpiryLevelLimit/kMaxLevelsPerPeriod directly?

Reply:
I think that is for unit tests purpose. Could you please add comments "For unit tests config"?

Reply:
To be able to run tests with different values. Testing with max level of 100 requires building dag of 100 levels. See the dag_expiry test I added in dag_test

Reply:
Could you please add comments?

uint32_t dag_expiry_limit = kDagExpiryLevelLimit;               // For unit tests only
uint32_t max_levels_per_period = kMaxLevelsPerPeriod;   // For unit tests only

Reply:
Done